sdk/src/test/java/android/net/http/AndroidHttpClient.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) 2015 Uber Technologies, Inc.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+ 
+package android.net.http;
+
+/**
+ * Since Apache HTTP Client was deprecated, it was no longer on the compile classpath. 
+ * However, Robolectric 3.0 still links to it. So we need to add no-op class for successful compiling.
+ * This workaround work only if we do not use AndroidHttpClient in our test cases. 
+ * 
+ * Link to Robolectric issue https://github.com/robolectric/robolectric/issues/1862
+ * 
+ * Created by Antonenko Viacheslav on 24/02/16.
```

Comment:
Can you remove this line?  Sorry, we don't have any attribution in our JavaDocs.